### PR TITLE
Custom email templates: remove background-image from list of attributes

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Removes `background-image` from the list of allowable attributes. Also, updates guide to bring Acrolinx score to 80.
- **Is this PR related to a Monolith release?** No.

### Resolves:

* [OKTA-661396](https://oktainc.atlassian.net/browse/OKTA-661396)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ The following table provides a list of all allowed HTML tags and elements in cus
...
...`, `height`, `image()`, `letter-spacing`, `line-height`, `linear-gradient()`, `list-style`, `list-style-image`, `list-style-position`, `list-style-type`, `margin`, `margin-bottom`, `margin-left`, `margin-right`, `margin-top`, `max-height`, `max-width`, `min-height`, `min-width`, `outline`, `outline-color`, `outline-style`, `outline-width`, `padding`, `padding-bottom`, `padding-left`, `padding-right`, `padding-top`,`pause`, `pause-after`, `pause-before`, `pitch`, `pitch-range`, `quotes`, `radial-gradient()`, `rect()`, `repeating-linear-gradient()`, `repeating-radial-gradient()`, `rgb()`, `rgba()`, `richness`, `speak`, `speak-header`, `speak-numeral`, `speak-punctuation`, `speech-rate`, `stress`, `table-layout`, `text-align`, `text-decoration`, `text-indent`, `text-overflow`, `text-shadow`, `text-transform`, `text-wrap`, `unicode-bidi`, `vertical-align`, `voice-family`, `volume`, `white-space`, `width`, `word-spacing`, `word-wrap`,`display`, `float`, `position`, `right`, `left`,`top`, `bottom`, `clear`, `content`, `cursor`, `z-index`, `overflow`, `visibility`, `zoom`, `overflow-x`, `overflow-y`, `page-break-inside`, `page-break-before`, `page-break-after`, `opacity`, `-moz-opacity` |
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:165]: @brianduffield-okta This list.

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md [hunk 2]
```diff
@@ -330,7 +330,7 @@ Variables used for function parameters must match the function data type. For ex
 | escapeHtml(String html)                                       | Escapes the characters in the provided string using HTML entities                                                             |
 | escapeHtmlAttr(String html)                                   | Encodes data for use in HTML attributes                                                                           |
 
-## Test custom email templates
+## Test your custom email templates
```
Comment (brianduffield-okta) [inline packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:333]: You might want to search on #test-custom-email-templates .... to see if you broke a cross-reference

**Review by brianduffield-okta — APPROVED** [review]
LGTM! One suggestion to search on heading change.

File: packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md (hunk 2 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:333]: Right... of course! Will do. Thank you.

Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/guides/custom-email/main/index.md:333]: @brianduffield-okta Turns out it's not an issue. No x-refs!

**Pull request merged**